### ISSUE TITLE
emit text input change event

### ISSUE THREAD

**Issue by foxhound87** [TotomInc/vue3-select-component#69]
Hello,
I would like to handle the text input change event when you type in the input field.

For example like this:

```
            <VueSelect
              isSearchable
              @search="onSelectSearch" <<---
              v-model="selected"
              :is-multi="true"
              :options="[
                { label: 'Option #1', value: 'option_1' },
                { label: 'Option #2', value: 'option_2' },
                { label: 'Option #3', value: 'option_3' },
              ]"
            />
```

```
function onSelectSearch(event) {
  console.log("onSelectSearch", event.target.value);
  // access here the text input value and trigger an HTTP call
}
```

Thank you!

**Comment by TotomInc** [comment]
Hi, thanks for reporting this issue.

I can add an event when the search value is updated.

However, is it possible to add a bit more details about your use-case?

**Comment by foxhound87** [comment]
thank you very much!

**Comment by TotomInc** [comment]
Will notice when this feature is deployed on a new version.

**Comment by TotomInc** [comment]
Available on v0.3.0

cc @foxhound87
